README.md

```diff
@@ -1,61 +1,191 @@
-Backend of the Alcohol Wholesale Registration Scheme application
-================================================================
+AWRS
+====
 
 [![Build Status](https://travis-ci.org/hmrc/awrs.svg)](https://travis-ci.org/hmrc/awrs) [ ![Download](https://api.bintray.com/packages/hmrc/releases/awrs/images/download.svg) ](https://bintray.com/hmrc/releases/awrs/_latestVersion)
 
-This service provides the backend endpoint for the [Alcohol Wholesale Registration Scheme project](https://github.com/hmrc/awrs-frontend).
+This service provides the backend endpoints for the [Alcohol Wholesale Registration Scheme Frontend project](https://github.com/hmrc/awrs-frontend), allowing a customer to apply for apply for the Alcohol Wholesale Registration Scheme.
```

Comment:
Typo in here? double reference to 'apply for'

Reply:
Well spotted - I've fixed it and pushed